src/test/java/org/testeditor/fixture/web/WebDriverFixtureLocalFirefoxTest.java

```diff
@@ -154,23 +140,26 @@ public void newFirefoxPortableStartAndStopTest() throws InterruptedException, IO
 	 */
 	//@Test
 	public void newFirefoxStartAndStopTest() throws InterruptedException, IOException {
+		
+		//GIVEN
```

Comment:
Usually we write `// Given`, `// When`, `// Then` or `// given`, `// when`, `// then`.